telegram/plugin.yaml

```diff
@@ -109,6 +109,12 @@ item_attributes:
             de: 'Ein Item dessen Datentyp dict ist und Paare von ``Chat_id`` der vertraut wird und Schreibzugriff ``True`` oder ``1``'
             en: 'An item of type dict with trusted ``chat_id`` and write access ``True`` or ``1``'
 
+    telegram_control:
+        type: str
```

Comment:
Type string does not seem correct here. It looks as if it was a list of strings from the sample above.